### PR TITLE
Adding Node Boost to Config

### PR DESCRIPTION
**Pull request by Xavbeat03**
Added the Node Boost Boolean to config as well as the getter for the value in TownyProductionSettings.java Name of value may have to be refined.

### CODE REVIEW

**Comment by Xavbeat03** [comment]
Things that need to be completed:
- [ ] Costs money for the duration of increased production
- [ ] Certain NodeTypes can have a flag "UNBOOSTABLE" that stops /node boost from being executed on them
- [ ] NodeTypes can also have the BOOST-MULT flag that multiplies the cost of boosting
- [ ] Can be disabled in config
- [ ] Testing

File: src/me/NinjaMandalorian/TownyProduction/object/ProductionNode.java [hunk 1]
```diff
@@ -27,6 +27,9 @@ public class ProductionNode {
 	private int nodeTier;
 	private ArrayList<WorldCoord> ClusterCoords;
 	private long lastCollected;
+	private long boostInitializationTime;
+	private final long boostDurationTime = 60; //change to a more reasonable value later on
+	private double nodeBoostPercentage = 0;
```
Comment (NinjaMandalorian) [inline src/me/NinjaMandalorian/TownyProduction/object/ProductionNode.java:32]: You only need to have the end date as a variable in ProductionNode, and the boost % will be a config setting.

**Review by NinjaMandalorian — CHANGES_REQUESTED** [review]
Good start, I can finish this off if you want? Whatever is easiest.

With UNIX, you only need to store the end value, as it doesn't really matter when it started.

**Comment by Xavbeat03** [comment]
The system that changes the item amounts is working well.
Catch for node already boosting is doing well.
Not sure how to check if PERMs is working, or the node flags, so that'll need to be tested later
Also not sure how to add the checks for money or subtracting it, gonna need to figure that out

**Comment by Xavbeat03** [comment]
Flags don't seem to be getting properly @NinjaMandalorian if you have a second, can you take a look and figure out why?

**Review by NinjaMandalorian — APPROVED** [review]
Tested and working.

**Pull request merged**